app/controllers/RemoveSavedDeclarationsController.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import config.AppConfig
+import connectors.CustomsDeclareExportsConnector
+import controllers.actions.AuthAction
+import forms.RemoveDraftDeclaration.form
+import javax.inject.Inject
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+import views.html.remove_declaration
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class RemoveSavedDeclarationsController @Inject()(
```

Comment:
Is not part of declarations controller ?

Reply:
Theres a "confirmation page" which I think this controller represents @bambuchaAdm